hack/debug.sh

```diff
@@ -249,16 +261,19 @@ do_pod_service_connectivity_check () {
 }
 
 do_node () {
-    config=$(ps wwaux | grep -v grep | sed -ne 's/.*openshift.*--config=\([^ ]*\.yaml\).*/\1/p')
+    # Find the config file
+    config=$(ps wwaux | grep -v grep | sed -ne 's/.*openshift.*--\(node-\)\?config=\([^ ]*\.yaml\).*/\1/p')
```

Comment:
of course, this (both before and after) has the same problem as the master code above, in reverse: if run on the master, and the master process was started with --config, then this might pick up the master config rather than the node config...